README.md

```diff
@@ -1,2 +1,197 @@
-# CTS2023
-It is my first capstone project in Microverse
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Climate Thecnlology Show 2023</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Climate Tachnology Show 2023] <a name="about-project"></a>
+
+**[Climate Tachnology Show 2023]** is my first capstone project in Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML5
+- CSS3
+- Git version control
+- Javascript
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.microverse.org/">HTML5</a></li>
+    <li><a href="https://www.microverse.org/">CSS3</a></li>
+    <li><a href="https://www.microverse.org/">Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer">VS CODE Live Server Extension</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Header]**
+- **[Hero]**
+- **[About]**
+- **[Responsive]**
+- **[Mobile menu]**
+
+In this project best coding practices is used
+
+## 🚀 Live Demo 
+For visiting the live demo click <a href="https://gh-reza.github.io/CTS2023/" name="live-demo">here</a>.
```

Comment:
- [ ] I kindly suggest that you ensure that all pages, links, and images of the deployed link work as expected. I clicked on your live deployed page and I got the result below:

<img width="960" alt="image" src="https://github.com/Gh-Reza/CTS2023/assets/52934128/8d3a1b84-e2b4-41f1-b3eb-d4959328e423">